cinnabar/plotlying.py

```diff
@@ -266,11 +269,18 @@ def _master_plot(
 
     # stats and title
     string = []
+    if statistic_type not in ['mle', 'mean']:
+        raise Exception(f"Unknown statistic type {statistic_type}")
```

Comment:
```suggestion
        raise ValueError(f"Unknown statistic type {statistic_type}")
```
Please don't raise a plain Exception, use a `ValueError` here? Also please add test for exception.